ci/docker/runtime_functions.sh

```diff
@@ -763,7 +763,7 @@ cd_unittest_ubuntu() {
     fi
 
     if [[ ${mxnet_variant} = *mkl ]]; then
```

Comment:
"*mkl" seems to be not valid from mxnet1.8  or 1.7?  but I am not sure if in this context also

Reply:
Not in this context.